### DISCUSSION TITLE
Merging into scummvm?

### DISCUSSION THREAD

**Discussion by rubyFeedback** [ihhub/fheroes2#5827] [General]
### Your question

Are there any plans to merge with scummvm?

**Comment by ihhub** [comment]
Hi @rubyFeedback , no, we won't do this.

**Reply by shprotru** [reply]
Hi @rubyFeedback the reason to merge into scummvm is may be dynamic screen resizing, that was done by developers of scummvm. Currently fheroes2 has own nice rendering engine which makes it self-sufficient project. Any other reasons?

**Comment by ihhub** [comment]
License is the only and the main reason. We have no rights to distribute the game.

**Reply by shprotru** [reply]
scummvm is games engine, not a game, it's not provide game resources.

**Comment by ihhub** [comment]
Who's going to update the engine and source code there? How to do the integration? What's the benefit of doing this instead of proving a separate application?
